db/migrate/20231113211138_add_esign_to_state_intakes.rb

```diff
@@ -0,0 +1,13 @@
+class AddEsignToStateIntakes < ActiveRecord::Migration[7.1]
+  def change
+    add_column :state_file_ny_intakes, :primary_esigned, :integer, default: 0, null: false
+    add_column :state_file_ny_intakes, :primary_esigned_at, :datetime
+    add_column :state_file_az_intakes, :primary_esigned, :integer, default: 0, null: false
+    add_column :state_file_az_intakes, :primary_esigned_at, :datetime
+
+    add_column :state_file_ny_intakes, :spouse_esigned, :integer, default: 0, null: false
+    add_column :state_file_ny_intakes, :spouse_esigned_at, :datetime
+    add_column :state_file_az_intakes, :spouse_esigned, :integer, default: 0, null: false
+    add_column :state_file_az_intakes, :spouse_esigned_at, :datetime
```

Comment:
I think we can do this with datetimes only, but I'd want to double check against prior signature implementations.